### PR TITLE
haydn: update v11.0

### PR DESCRIPTION
**Pull request by INCENDIOCODM**

### CODE REVIEW

File: haydn.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-			"sha256": "b0efc366c53aafc0a4554a5ab08ce7d992402e9b8f726ba85ed6de55097fd8fe",
-			"size": 2046475507,
+			"filename": "crDroidAndroid-15.0-20241217-haydn-v11.0.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/haydn/11.x/crDroidAndroid-15.0-20241217-haydn-v11.0.zip/download",
+			"timestamp": 1734407774,
+			"md5": "7c8b84a874de8ca18bc19a0c906ed70c",
+			"sha256": "0d6c537cf49beedba0b99fd209427d58a3aa313838c99ba64beae218792db6c4",
+			"size": 2079887797,
 			"version": "11.0",
 			"buildtype": "BETA",
```
Comment (gwolf2u) [inline haydn.json]: out of beta

File: haydn_changelog.txt
Comment (gwolf2u) [inline haydn_changelog.txt:1]: also change build type here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: haydn_changelog.txt
Comment (INCENDIOCODM) [reply haydn_changelog.txt:1]: Done

Comment (gwolf2u) [inline haydn_changelog.txt:1]: Stil beta in changelog

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Review by gwolf2u — APPROVED** [review]

File: haydn_changelog.txt
Comment (INCENDIOCODM) [reply haydn_changelog.txt:1]: I'm sorry,  i was on phone, it didn't show the changelog file 
now its fixed

**Pull request merged**